### ISSUE TITLE
If bin edge is too large, throw an informative error. 

### ISSUE THREAD

**Issue by mclaughlin6464** [manodeep/Corrfunc#62]
I was using corrfunc and I ran into this error:

`linklist> ERROR:  nlattice = 2 is so small that with periodic wrapping the same cells will be counted twice ....exiting`

Which was a bit hard to parse. I eventually pieced together that the binfile I had written contained a bin edge that was too large relative to the size of my box. It would be helpful is the error message could say this, rather than sending me to the source code.

**Comment by manodeep** [comment]
Thanks for reporting the issue! In commit 9be3a37, I have updated the relevant files to be more explicit about the error -- please let me know if the resolution works for you.

**Comment by manodeep** [comment]
Fixed the incorrect commit message. @mclaughlin6464 already confirmed that the fix worked - closing this issue.
